cmd/query/app/server.go

```diff
@@ -329,10 +329,7 @@ func (s *Server) Close() error {
 	errs = append(errs, s.queryOptions.TLSHTTP.Close())
 	s.grpcServer.Stop()
 	errs = append(errs, s.httpServer.Close())
-	if s.separatePorts {
-		errs = append(errs, s.httpConn.Close())
-		errs = append(errs, s.grpcConn.Close())
-	} else {
```

Comment:
closing the servers already closes the underlying connections. When killing the binary it was logging errors like:
```
Error: failed to shutdown service after error: failed to shutdown extensions: closing httpConn: close tcp [::]:16686: use of closed network connection
```